site/docs/v1/tech/guides/breached-password-detection.adoc

```diff
@@ -0,0 +1,306 @@
+
+Let's examine each of these in turn.
+
+=== Match Mode
+
+When FusionAuth is checking for compromised credentials, there are a number of ways to attempt to match them. Assume you have a user with the username `richard@piedpiper.com` and they are trying to sign up with the password `This333ABCpassword!`.
+
+The system could match on the password string (`This333ABCpassword!`) only. If the password was in the dataset and any other user had ever used it, FusionAuth would consider this a match and not allow this password.
+
+Another way of matching would be to see if there are any sub-email accounts associated with that password. These are also called aliases by some email providers such as gmail. For instance, `richard+test@piedpiper.com` is an alias of `richard@piedpiper.com`. The system could check multiple addresses, such as `richard+test@piedpiper.com` and `richard+foo@piedpiper.com`. All of these addresses would be paired with the provided password and checked against the datasets. 
+
+If you match on sub-email accounts, if the pair of `richard+test@piedpiper.com` and `This333ABCpassword!` was found in the corpus, FusionAuth would consider this a match and not allow this password. On the other hand, if `jared@piedpiper.com` and `This333ABCpassword!` were in the dataset, `richard@piedpiper.com` would still be allowed to use this password.
+
+You could also narrow your scope further and match on only the username/password pair. If `richard@piedpiper.com` and `password!` appeared together in the password list, then FusionAuth would not allow this password. If `richard+test@piedpiper.com` and `This333ABCpassword!` or `jared@piedpiper.com` and `This333ABCpassword!` were, the password would be allowed.
+
+You can control which level of matching will disallow use of a password. This is called the *Match mode* configuration setting and can be one of the following values:
+
+* `High` means that should any of the above situations hold, the check fails. This is the strictest option.
+* `Medium` means that an exact username and password match, a commonly compromised password is found, or a sub-email address match disqualifies the password.
+* `Low` means that an exact username and password match or a commonly compromised password results in a failed check.
+
+Assume the dataset contains `richard@piedpiper.com` with a password of `This333ABCpassword!`.com` with the same password. If the following users tried to register, the results for a given mode would be:
+
+.Match modes and what is allowed
+|===
+| Username | Password | High | Medium | Low 
+
+|`richard@piedpiper.com`
+|`This333ABCpassword!`
+|&#10060;
+|&#10060;
+|&#10060;
+
+|`richard+test@piedpiper.com`
+|`This333ABCpassword!`
+|&#10060;
+|&#10060;
+|&#9989;
+
+|`jian@piedpiper.com`
+|`This333ABCpassword!`
+|&#10060;
+|&#9989;
+|&#9989;
+
+|`richard@piedpiper.com`
+|`ADifferent333pass!`
+|&#9989;
+|&#9989;
+|&#9989;
```

Comment:
Not a big deal, but we could use font awesome here for similarity between our UI and docs.

Reply:
Good point!